lib/screens/explore_screens/vouch_for_a_member/interactor/viewmodel/vouch_for_a_member_event.dart

```diff
@@ -0,0 +1,33 @@
+part of 'vouch_for_a_member_bloc.dart';
+
+abstract class VouchForAMemberEvent extends Equatable {
+  const VouchForAMemberEvent();
+
+  @override
+  List<Object> get props => [];
+}
+
+class OnUserSelected extends VouchForAMemberEvent {
+  final MemberModel user;
+
+  const OnUserSelected(this.user);
+
+  @override
+  String toString() => 'OnUserSelected: { User: $user }';
+}
+
+class OnConfirmVouchForMemberTab extends VouchForAMemberEvent {
+  final MemberModel user;
```

Comment:
you have the selected user in the state, no need to pass around. 
In the dialog, you can grab it from the state. 
When this is pressed, you can grab it form the state

Reply:
fix